src/client/datascience/ipywidgets/commonMessageCoordinator.ts

```diff
@@ -68,6 +70,7 @@ export class CommonMessageCoordinator {
             }>();
         this.disposables = this.serviceContainer.get<IDisposableRegistry>(IDisposableRegistry);
         this.jupyterOutput = this.serviceContainer.get<IOutputChannel>(IOutputChannel, STANDARD_OUTPUT_CHANNEL);
+        this.appSell = this.serviceContainer.get<IApplicationShell>(IApplicationShell, IApplicationShell);
```

Comment:
Typo wasn't fixed here, but we can fix just in main.

Reply:
Oops, I must have missed that in the other review, will fix in main.